sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Tests.csproj

```diff
@@ -1,15 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit" />
-    <PackageReference Include="xunit.runner.visualstudio" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
```

Comment:
@vishweshbankwar If I recall correctly, Moq was removed from the OpenTelemetry .NET SDK. Do you know if that's correct, and can you explain the reason? Should we avoid using it here?

Reply:
> if I recall correctly, Moq was removed from the OpenTelemetry .NET SDK

That is correct.

I dont see its use in tests. do we need it? Ideally we should not need to use Moq for anything, if there is a need we should first try to refactor the code.